lambdas/email_appointments.py

```diff
@@ -0,0 +1,214 @@
+import json
```

Comment:
Is this an edit to appointment_email.py? If so, please rename this back to the old file name, so I can see what changes you made.

Reply:
Yes it is an edit. I will do that.